src/Resources/skeleton/crud/templates/show.tpl.php

```diff
@@ -0,0 +1,23 @@
+<?= $helper->getHead($base_layout_exists, $entity_class_name) ?>
+
+{% block body %}
+
+    <h1><?= $entity_class_name ?></h1>
+
+    <table>
+<?php foreach ($entity_fields as $field): ?>
+        <tr>
+            <th><?= ucfirst($field['fieldName']) ?></th>
+            <td>{{ <?= $helper->getEntityFieldPrintCode($entity_var_singular, $field) ?> }}</td>
+        </tr>
+<?php endforeach; ?>
+
+    </table>
+
+    <a href="{{ path('<?= $route_name ?>_index') }}">back to list</a>
+
+    <a href="{{ path('<?= $route_name ?>_edit', {'<?= $entity_identifier ?>': <?= $entity_var_singular ?>.<?= $entity_identifier ?>}) }}">edit</a>
+
+    {% include '<?= $route_name ?>/_delete_form.html.twig' with {'identifier': <?= $entity_var_singular ?>.<?= $entity_identifier ?>} only %}
```

Comment:
Any special reason to take the effort to specifically pass in the `identifier` variable AND use the `only` part? I mean it's more explicit, but looks more complex. But I thought you might also have a specific reason.

Reply:
Just thought about some template optimization. I don't think that we must pass whole context to template which doesn't use it at all